apps/ecs/translations/src/en/fields.json

```diff
@@ -35,5 +35,36 @@
         "label": "No"
       }
     }
+  },
+  "worker-applied-eu-settlement-scheme": {
+    "legend-1": "Has the worker applied to the",
+    "legend-link": "EU Settlement Scheme",
+    "legend-2": "(EUSS)?",
```

Comment:
I think its better to move this legend text to pages.json as we have contents usually in pages.json for all other pages.